### ISSUE TITLE
Error: ENOENT: no such file or directory

### ISSUE THREAD

**Issue by TurboTobiasDK** [henrikwirth/gatsby-starter-wordpress-advanced#3]
Having issues running gatsby build with starter theme. Seems like there is some path issue:

C:\Users\Tobias\AppData\Roaming\npm\node_modules\gatsby-cli\node_modules\yoga-layout-prebuilt\yoga-layout\build\Release\nbind.js:53
throw ex;
^
Error writing tmp-/ template: Error: ENOENT: no such file or directory, open 'C:\Users\Tobias\Desktop\Gatsby\gatsby-starter-wordpress-advanced.template-cache\tmp-.js'

I've also tried with gatsby clean to clear the cache. I have also tried deleting .template-cache folder & node_module folder and ran npm install. The issue still remains.

**Comment by henrikwirth** [comment]
Yeah I think thats an issue with Windows paths. Although it seems weird in your case, as there is no uri after the `tmp-` at all.

What WPGrahpQL version are you using? Did you adjust anything in the setup?

**Comment by TurboTobiasDK** [comment]
WordPress running on v5.4 using Local by Flywheel. Fetching data correctly using .env file as described in your blog post.

Installed plugins:
Advanced Custom Fields PRO v5.8.9
Classic Editor v1.5
WP GraphiQL v1.0.1
WP GraphQL v0.8.1
WPGraphQL for Advanced Custom Fields v0.3.1

I've follow your blog post tutorial and everything went very well until part 7 where the ACF Page Builder was being added to the project. I have made some minor changes. I changed post url slug from /blog/post-name/ to just /post-name/.

See my repo: https://github.com/TurboTobiasDK/bp-gatsby-wpgraphql

**Comment by henrikwirth** [comment]
Hi @TurboTobiasDK I found the solution. I setup my Windows PC to test it and it works now for me.

First of all, I had to update the paths, because the page.uri is different in the newer versions of WPGraphQL. Before it was just: `some-page` now it is `/some-page/`.

Secondly in the page template creation process I was using the uri, therefore, this messed up the paths for the template files. I switched to page.slug instead now and did some extra checks, to make sure the frontPage is not ending up producing a wrong path.

Just checkout the Master-branch. I also updated the Tutorials already. Let me know if it works for you, so I can close the issue.
